test/e2e/framework/pv_util.go

```diff
@@ -233,15 +290,18 @@ func CreatePVCPV(c clientset.Interface, pvConfig PersistentVolumeConfig, pvcConf
 
 	By(fmt.Sprintf("Creating a PVC followed by a%s PV", preBindMsg))
 	// instantiate the pvc
-	pvc = CreatePVC(c, ns, pvc)
+	if pvc, err = CreatePVC(c, ns, pvc); err != nil {
```

Comment:
To clarify, I meant the style
```golang
var x int
...
if x, err = f(); err != nil {
```
IMO it's clearer and easier to read when we split the call and the if:
```golang
x, err = f()
if err != nil {
```

But, this is hairsplitting and just taste. Leave it as it is.

I am more dogmatic about declaring variables at the top and reuse them (if not absolutely necesserary). For example in CreatePVsPVCs 
 below (https://github.com/kubernetes/kubernetes/pull/43956/files#diff-420013911dac4a3b0f16035b6e90a571R343): pv, pvc, err are declared at the top, but there is no need to persist values over loop iterations. Hence, delcare those variables locally inside the loops. Never reuse them (maybe with the exception of `err`). The compiler can optimize the code to be efficient about stack usage.

In contrast, pvMap and pvcMap in the same func are global for the whole function scope. Having those at the top is absolutely fine.